composer.json

```diff
@@ -10,7 +10,8 @@
         "source": "https://cgit.drupalcode.org/message_notify"
     },
     "require": {
-      "drupal/message": "~8.1.0"
+        "drupal/message": "~8.1.0",
+        "drupal/sms": "^1.1"
```

Comment:
sms should be in `suggest`, not `require`.

Use require-dev if needing sms framework code for testing.